Projects/Presentation/Sources/Company/CompanyDetailViewController.swift

```diff
@@ -106,7 +106,7 @@ class CompanyDetailViewController: BaseViewController<RecruitmentViewModel> {
         scrollView.snp.makeConstraints {
             $0.top.equalTo(self.view.safeAreaLayoutGuide)
             $0.left.right.equalTo(self.view.safeAreaLayoutGuide)
```

Comment:
```suggestion
            $0.leading.trailing.equalTo(self.view.safeAreaLayoutGuide)
```
left,right 말고 leading,trailing 써주세요